reference/component-versions.md

```diff
@@ -2,6 +2,68 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+## 1.21.2
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.21/stable` | 307 |
+| `ams-node-controller` | `1.21/stable` | 315 |
+| `coturn` | `1.21/stable` | 309 |
+| `ams` | `1.21/stable` | 320 |
+| `aar` | `1.21/stable` | 314 |
+| `anbox-stream-gateway` | `1.21/stable` | 325 |
+| `ams-lxd` | `1.21/stable` | 312 |
+| `anbox-stream-agent` | `1.21/stable` | 320 |
+| `nats ` | `latest/stable` | 11 |
+
+#### Ubuntu 20.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.21/stable` | 306 |
+| `ams-node-controller` | `1.21/stable` | 314 |
+| `coturn` | `1.21/stable` | 308 |
+| `ams` | `1.21/stable` | 319 |
+| `aar` | `1.21/stable` | 313 |
+| `anbox-stream-gateway` | `1.21/stable` | 324 |
+| `ams-lxd` | `1.21/stable` | 311 |
+| `anbox-stream-agent` | `1.21/stable` | 319 |
+| `nats ` | `latest/stable` | 11 |
+
+### Bundles
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud` | `1.21/stable` | 182 |
+| `anbox-cloud-core` | `1.21/stable` | 189 |
+
+### Snaps
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.21/stable` | 1.21.2 |
+| `aar`    | `1.21/stable` | 1.21.2 |
+| `ams-node-controller` | `1.21/stable` | 1.21.2 |
+| `anbox-cloud-dashboard` | `1.21/stable` | 1.21.2 |
+| `anbox-stream-agent` | `1.21/stable` | 1.21.2 |
+| `anbox-stream-gateway` | `1.21/stable` | 1.21.2 |
+| `anbox-cloud-appliance` | `1.21/stable` | 1.21.2 |
+| `anbox-cloud-tests` | `1.21/stable` | 1.21.2 |
+
+### Anbox images
+
+| Name | Version |
+|----------|--------------|
+| `jammy:android13:amd64` | 1.21.2 |
+| `jammy:android13:arm64` | 1.21.2 |
+| `jammy:android12:amd64` | 1.21.2 |
+| `jammy:android12:arm64` | 1.21.2 |
```

Comment:
I'm sorry for not noticing in previous iterations, but shouldn't we also have the AAOS images there?
These would be:
```suggestion
| `jammy:android13:amd64` | 1.21.2 |
| `jammy:android13:arm64` | 1.21.2 |
| `jammy:android12:amd64` | 1.21.2 |
| `jammy:android12:arm64` | 1.21.2 |
| `jammy:aaos13:amd64` | 1.21.2 |
| `jammy:aaos13:arm64` | 1.21.2 |
```

Reply:
Oh yes, I have included it for previous releases as well (dating back to 1.21.0). But I feel it is better to distinguish with an indicator to show that these are still developing. I have used *Experimental* for now but wondering if something else like *Beta* would be more professional.

Reply:
Decided to use experimental based on discussion in SU